src/systems/a320_systems/src/hydraulic.rs

```diff
@@ -2644,6 +3005,73 @@ mod tests {
             assert!(test_bed.yellow_pressure() < Pressure::new::<psi>(500.));
         }
 
+        #[test]
+        fn yellow_edp_solenoid_unpowered() {
```

Comment:
**issue**: As in the previous PR, these names indicate what you are testing.
**suggestion**: Use function names that describe what you are testing, e.g.:
`when_yellow_edp_solenoid_unpowered_yellow_hydraulic_system_is_pressurised`